Tiltfile.dev

```diff
@@ -0,0 +1,17 @@
+local_resource(
+  name = 'dev:acsp-api',
+  cmd = 'mvn clean compile',
+  deps = ['src']
+)
+custom_build(
+  ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/acsp-api',
+  command = 'mvn clean compile jib:dockerBuild -Dimage=$EXPECTED_REF',
+  live_update = [
+    sync(
+      local_path = './target/classes',
+      remote_path = '/app/classes'
+    ),
+    restart_container()
+  ],
+  deps = ['./target/classes']
+)
```

Comment:
Also might be worth checking if you need the lines which reference this within this file. 

https://github.com/companieshouse/presenter-account-api/blob/main/Tiltfile.dev#L20-#L26

Reply:
@Ngriffiths1 ran this via lead dev, toggle not need at present hence not included.